common/src/main/scala/com/aliyun/emr/rss/common/protocol/message/ControlMessages.scala

```diff
@@ -758,14 +760,10 @@ object ControlMessages extends Logging{
       case GET_REDUCER_FILE_GROUP_RESPONSE =>
         val pbGetReducerFileGroupResponse = PbGetReducerFileGroupResponse
           .parseFrom(message.getPayload)
-        val fileGroup = if (pbGetReducerFileGroupResponse.getFileGroupCount > 0) {
-          pbGetReducerFileGroupResponse.getFileGroupList.asScala
-            .map(fg => fg.getLocaltionsList.asScala
-              .map(PartitionLocation.fromPbPartitionLocation(_)).toArray).toArray
-        } else null
-        val attempts = if (pbGetReducerFileGroupResponse.getAttemptsCount > 0) {
-          pbGetReducerFileGroupResponse.getAttemptsList().asScala.map(Int.unbox(_)).toArray
-        } else null
+        val fileGroup = pbGetReducerFileGroupResponse.getFileGroupList.asScala.map { fg =>
+            fg.getLocaltionsList.asScala.map(PartitionLocation.fromPbPartitionLocation).toArray
```

Comment:
`Int.unbox` should be same as `toInt`?

Reply:
I believe so

Reply:
There is another unbox when process COMMIT_FILES below.

Reply:
Has been handled in https://github.com/alibaba/RemoteShuffleService/pull/237/